cmd/katapult/output_test.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"bytes"
+	"errors"
+	"testing"
+
+	"github.com/krystal/katapult-cli/internal/golden"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGenericOutput_JSON(t *testing.T) {
+	g := genericOutput{item: "hello world!"}
+	b, err := g.JSON()
+	assert.NoError(t, err)
+	assert.Equal(t, []byte(`"hello world!"`), []byte(b))
+}
+
+func TestGenericOutput_YAML(t *testing.T) {
```

Comment:
This test function should be called `Test_genericOutput_YAML` as `genericOutput` starts with a lowercase letter.

Also I think it should be a tabular test that verifies expected output for a few different types of input, to ensure it does correctly marshal whatever it's given as input to YAML.